src/pages/genesis.tsx

```diff
@@ -117,18 +116,15 @@ export default function IndexPage() {
     }
 
     if (isFilteringByCategory && !params.only_view_category) {
-      const categoriesFetch = []
-      // TODO: review later. use map instead
-      for (const category of params.categories) {
-        const placesFetch = Places.get().getPlaces({
+      const categoriesFetch = params.categories.map((category) => {
```

Comment:
is this having a good performance?

Reply:
yep, it's been working in this way since its release.